pkg/account/handler.go

```diff
@@ -0,0 +1,167 @@
+package account
+
+import (
+	"container/list"
+	"fmt"
+	"math/rand"
+	"net/http"
+	"regexp"
+	"strconv"
+	"time"
+
+	"github.com/labstack/echo/v4"
+)
+
+var account_list = list.New()
+
+type AccountHandler struct {
+}
+
+/**
+ * @brief create and account based on email(as id), type, name and password
+ * @desc will check primarykey other, then add to account_list if possible
+ */
+func (h *AccountHandler) CreateAccount(c echo.Context) error {
```

Comment:
You dont't have to specify an implementation now, but that's ok if you've done that